CHANGELOG.md

```diff
@@ -5,59 +5,41 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
-- `\grechangedim` updated to reflect string storage of distances
-- User settable distances now stored as strings (see [#50](https://github.com/gregorio-project/gregorio/issues/50))
-- `\gre@stafflinefactor` now uses same scale as `\grefactor`
-- Improved `\includescore` to remove outdated files ([#61](https://github.com/gregorio-project/gregorio/issues/61)).
-- TeXworks configuration script for Windows updated
+- Unify version numbers. The `gregoriotex_api_version` has been deprecated. The only version number is the gregorio release version. This simplifies the versioning process and should make bug reports easier to address.
+- Distances in default space configuration (`gsp-default.tex`) have been changed so that they represent the actual printed dimension at the default staff size (see [#50](https://github.com/gregorio-project/gregorio/issues/50)).
+- User settable distance names have been simplified by removing the `\gre` prefix.  See doc/UserManual.pdf section ?? for a full list of the distance names.
+- Possibility to switch on/off scaling of distances with staff size, as a consequence, it is now possible to define distances in em units (scaling with text font), see doc/UserManual.pdf section ?? for details.
+- `\grechangedim` now takes three arguments: the name of the distance, value to change the distance to, and whether or not this value should be scaled with changes in the staff size.
+- `\setinitalspacing` now takes four arguments.  New argument specifies whether distances should scale when the staff size changes.
+- `\setspacebeforeinitial`, `\setspaceafterinitial`, and `\setaboveinitialseparation` now take two arguments.  The new argument specifies whether the distance should scale when the staff size changes.
+- Improved `\includescore` capabilities.  See doc/UserManual.pdf section ?? for full details.
+- `\gre@stafflinefactor` (the internal control for the thickness of the stafflines) now uses same scale as `\grefactor`.
 - Windows installer documentation updated
-- Clivis stem length now follow Solesme's books conventions (see [#31](https://github.com/gregorio-project/gregorio/issues/31)).
-- Space Configuration Loading code refactored
-- Most distance calculations disentangled so that each calculate functions do not call each other (helps to avoid circularity)
-- `\grefactor` is now initialized at 17 (the default value).  Necessitated that most distances be rescaled (see [#50](https://github.com/gregorio-project/gregorio/issues/50)).
-- Distances renamed to `\gre@...`  Functions which calculate distances renamed to `\gre@calculate@...`
-- All code which defines or calculates a space (and nothing else) now reside in gregoriotex-spaces.tex
+- Clivis stem length now follow Solesmes' books conventions (see [#31](https://github.com/gregorio-project/gregorio/issues/31)).
+- TeXworks configuration script for Windows updated.  `greg-book` and `gregorio` engines will no longer be added.
```

Comment:
Why not. Also, why only under Windows? Are the engines still relevant for Linux and OSX? I'm asking that because I'm currently fixing `make dist`, and I'm blocked by this (the Makefiles are not up-to-date, and I'm not sure what to remove)